30-indoor-air-quality-and-gas-leak/firmware/README.md

```diff
@@ -0,0 +1,51 @@
+# Indoor Air Quality and Gas Leak Firmware
+
+The Indoor Air Quality and Gas Leak project’s firmware is built with [Platform.io](https://platformio.org/) 
+as an Arduino application. To run the firmware, start by ensuring you have 
+the prerequisites below installed.
+
+## Prerequisites
+
+1. Download and install [Visual Studio Code](https://code.visualstudio.com/).
+1. Install the [PlatformIO IDE extension](https://marketplace.visualstudio.com/items?itemName=platformio.platformio-ide)
+via the Extensions menu of Visual Studio Code.
+![Visual Studio Code extensions menu with a search of "platformio"](../../images/platformio-extension.png)
```

Comment:
```suggestion
![Visual Studio Code extensions menu with a search of "platformio"](../images/platformio-extension.png)
```

Reply:
Actually all the images in here are broke in the same way, so you should do a quick replace on all of them.